assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.test.js

```diff
@@ -811,4 +816,113 @@ describe( 'WidgetAreaRenderer', () => {
 			)
 		).toMatchSnapshot();
 	} );
+
+	describe( 'Error boundary', () => {
+		const mockTrackEvent = jest.spyOn( tracking, 'trackEvent' );
+		mockTrackEvent.mockImplementation( () => Promise.resolve() );
+
+		// eslint-disable-next-line no-console
+		const originalConsoleError = console.error;
+
+		// Suppress console error while we are testing error boundary.
+		beforeAll( () => {
+			// eslint-disable-next-line no-console
+			console.error = () => true;
+		} );
+
+		// Restore console error.
+		afterAll( () => {
+			// eslint-disable-next-line no-console
+			console.error = originalConsoleError;
+		} );
```

Comment:
We do not need to mutate the console in this way. We can simply use `expect( console ).toHaveErrored()` for the tests to pass.